src/storage/examples/src/lib.rs

```diff
@@ -136,6 +139,16 @@ pub async fn run_bucket_examples(buckets: &mut Vec<String>) -> anyhow::Result<()
     set_public_access_prevention_enforced::sample(&client, &id).await?;
     tracing::info!("running get_public_access_prevention example");
     get_public_access_prevention::sample(&client, &id).await?;
+    tracing::info!("running view_versioning_status example");
+    view_versioning_status::sample(&client, &id).await?;
+    tracing::info!("running enable_versioning example");
+    enable_versioning::sample(&client, &id).await?;
+    tracing::info!("running view_versioning_status example");
+    view_versioning_status::sample(&client, &id).await?;
+    tracing::info!("running disable_versioning example");
+    disable_versioning::sample(&client, &id).await?;
+    tracing::info!("running view_versioning_status example");
+    view_versioning_status::sample(&client, &id).await?;
```

Comment:
comment: I might not have only done one `view_versionsing_status`, but the extras are harmless.